<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="Tab">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>791</width>
      <height>581</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_1">
     <attribute name="title">
      <string>data_variables</string>
     </attribute>
     <widget class="QListWidget" name="xtest_list">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>70</y>
        <width>131</width>
        <height>321</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>x데이터의 특성</string>
      </property>
     </widget>
     <widget class="QListWidget" name="ytest_list">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>70</y>
        <width>131</width>
        <height>321</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>10</y>
        <width>261</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>y데이터의 특성</string>
      </property>
     </widget>
     <widget class="QListWidget" name="x_selected_view">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>70</y>
        <width>131</width>
        <height>321</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
     </widget>
     <widget class="QListWidget" name="y_selected_view">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>70</y>
        <width>131</width>
        <height>321</height>
       </rect>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="x_right_btn">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>410</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>right</string>
      </property>
     </widget>
     <widget class="QPushButton" name="y_left_btn">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>420</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>left</string>
      </property>
     </widget>
     <widget class="QPushButton" name="y_right_btn">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>420</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>right</string>
      </property>
     </widget>
     <widget class="QPushButton" name="x_left_btn">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>410</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>left</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>data_method</string>
     </attribute>
     <widget class="QRadioButton" name="all_radio_btn">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>130</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>전체 데이터</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="zero_radio_btn">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>180</y>
        <width>261</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0을 제외한 전체 데이터</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="cell_radio_btn">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>270</y>
        <width>251</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0을 제외한 cell 데이터</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="cell_radio_btn_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>360</y>
        <width>251</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>0을 제외한 cell 데이터 3x3</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>choose_train_method</string>
     </attribute>
     <widget class="QRadioButton" name="linear_radio_btn">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>110</y>
        <width>130</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>선형회귀</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="kernel_radio_btn">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>200</y>
        <width>130</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>커널회귀</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="tree_radio_btn">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>300</y>
        <width>130</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>트리회귀</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="dnn_radio_btn">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>130</y>
        <width>130</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>DNN</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="random_radio_btn">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>350</y>
        <width>151</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>랜덤포레스트</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rnn_radio_btn">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>190</y>
        <width>130</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>RNN</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="lstm_radio_btn">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>250</y>
        <width>130</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>LSTM</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="neural_regression_radio_btn">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>410</y>
        <width>130</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>신경망회귀</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="gauss_radio_btn">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>250</y>
        <width>130</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>가우스회귀</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="gru_radio_btn">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>310</y>
        <width>130</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>GRU</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>회귀</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>60</y>
        <width>91</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>딥러닝</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="svr_radio_btn">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>160</y>
        <width>130</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>SVR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="exit_btn">
      <property name="geometry">
       <rect>
        <x>680</x>
        <y>490</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>실행</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="conv1_radio_btn">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>380</y>
        <width>130</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>cnn</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
